heron/healthm/tests/java/com/twitter/heron/healthmgr/common/PackingPlanProviderTest.java

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.healthmgr.common;
+
+import java.util.HashSet;
+
+import com.microsoft.dhalion.events.EventManager;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.twitter.heron.healthmgr.common.HealthManagerEvents.TopologyUpdate;
+import com.twitter.heron.packing.roundrobin.RoundRobinPacking;
+import com.twitter.heron.proto.system.PackingPlans;
+import com.twitter.heron.spi.packing.PackingPlan;
+import com.twitter.heron.spi.statemgr.SchedulerStateManagerAdaptor;
+import com.twitter.heron.spi.utils.PackingTestUtils;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
```

Comment:
I've had this issue too. For some reason checkstyle doesn't always seem to fire on mac and I haven't been able to figure out why.